homework/Vasili_Zasinets/ShopTask/src/test/java/com/tutrit/java/quickstart/homework/shop/ShopTest.java

```diff
@@ -0,0 +1,47 @@
+package com.tutrit.java.quickstart.homework.shop;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigDecimal;
+
+public class ShopTest {
+    ProductRepository bank;
+
+    Shop myTestShop = new Shop();
+    Logger log = LoggerFactory.getLogger("Test ");
+
+    @Before
+    public void setUp() {
+        this.bank = new ProductRepository();
+        myTestShop.addProduct(bank.milk);
+        myTestShop.addProduct(bank.water);
+        myTestShop.addProduct(bank.bread);
+        myTestShop.addProduct(bank.cola);
+        myTestShop.addProduct(bank.meal);
```

Comment:
and if you did as I suggested above, then here you can have only `getBaseProductCart()`